src/applications/claims-status/tests/components/OverviewPage.unit.spec.jsx

```diff
@@ -217,27 +172,6 @@ describe('<OverviewPage>', () => {
     });
   });
 
-  it('should not render timeline without a phase', () => {
```

Comment:
LH doesn't have a `phase` key, so just removing this test instead of trying to retrofit it to LH